beginner.md

```diff
@@ -0,0 +1,588 @@
+# delite - creating custom components
+
+## delite background
+`delite` is a new JavaScript library born out of the [Dojo Toolkit Dijit framework](http://dojotoolkit.org/reference-guide/1.10/dijit).
```

Comment:
I've add comments from people being afraid bringing too much history from dijit in there. Plus dijit was more than delite (contains deliteful part as well). So I think it might be reasonable to keep this out of the tutorial scope (this does not help on how to use delite). If you want to keep it inside maybe as foot note instead of the very first thing in the tutorial.